lib/ansible/modules/messaging/kafka_topics.py

```diff
@@ -0,0 +1,384 @@
+                return self.module.fail_json(msg)
+        else:
+            cmd = ('%s --alter --if-exists --zookeeper %s '
+                   '--partitions %s --topic %s') % (self.executable,
+                                                    self.zookeeper,
+                                                    self.partitions,
+                                                    ''.join(self.topic))
+
+        try:
+            env = ''
+            if self.jaas_auth_file:
+                env = {'KAFKA_OPTS': self.kafka_env_opts}
+
+            if int(get_partitions) < int(self.partitions):
+                return self.module.run_command(cmd, environ_update=env)
+            else:
+                err_msg = """
+                The number of partitions for a topic can only be increased.
+                """
+                return self.module.fail_json(msg=to_native(err_msg))
+
+        except KafkaError as exc:
+            self.module.fail_json(msg=to_native(exc))
+
+    def create(self):
+        ''' Create new kafka topic. '''
+
+        # create action supports only 1 topic
+        if len(self.topic) > 1:
+                msg = 'alter action supports only 1 topic'
+                return self.module.fail_json(msg)
+        else:
+            if self.config:
+                config = ' --config '.join(
+                    "%s=%r" % (key, value) for (key, value) in self.config.items())
+
+                cmd = ('%s --create --if-not-exists --zookeeper %s '
+                       '--replication-factor %s --partitions %s '
+                       '--topic %s --config %s') % (self.executable,
+                                                    self.zookeeper,
+                                                    self.replication_factor,
+                                                    self.partitions,
+                                                    ''.join(self.topic),
+                                                    config)
+
+            else:
+                cmd = ('%s --create --if-not-exists --zookeeper %s '
+                       '--replication-factor %s --partitions %s '
+                       '--topic %s') % (self.executable,
+                                        self.zookeeper,
+                                        self.replication_factor,
+                                        self.partitions,
+                                        ''.join(self.topic))
+
+        try:
+            env = ''
+            if self.jaas_auth_file:
+                env = {'KAFKA_OPTS': self.kafka_env_opts}
+
+            return self.module.run_command(cmd, environ_update=env)
+
+        except KafkaError as exc:
+            self.module.fail_json(msg=to_native(exc))
+
+    def delete(self):
+        ''' Delete a kafka topic. '''
+
+        # create action supports only 1 topic
+        if len(self.topic) > 1:
+                msg = 'alter action supports only 1 topic'
```

Comment:
In the comment above and in the `msg` variable: s/create/delete/ s/alter/delete/.